tsuru/client/deploy.go

```diff
@@ -278,10 +289,75 @@ func (c *AppDeploy) Run(context *cmd.Context, client *cmd.Client) error {
 	return cmd.ErrAbortCommand
 }
 
-func targz(ctx *cmd.Context, destination io.Writer, filepaths ...string) error {
+func processTsuruIgnore(pattern string, dirPath ...string) ([]string, error) {
+	var paths []string
+	old, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+	defer os.Chdir(old)
+	for _, dir := range dirPath {
+		err = os.Chdir(dir)
+		if err != nil {
+			return nil, err
+		}
+		wd, err := os.Getwd()
+		if err != nil {
+			return nil, err
+		}
+		paths, err = filepath.Glob(pattern)
+		if err != nil {
+			return nil, err
+		}
+		for i := range paths {
+			if dir == "." {
```

Comment:
It is, we got this as a bug while uploading `static` into lab so without It, the path would be only `paths[i]` and It would never match `glob`